scss/3-generics/_fonts.scss

```diff
@@ -2,39 +2,26 @@
 // Fonts
 // ============================================================================
 
+/* open-sans-regular - latin */
 @font-face {
   font-family: 'Open Sans';
   font-style: normal;
   font-weight: 400;
-  font-display: swap;
-  src: url('../../fonts/open-sans-v17-latin-regular.eot'); /* IE9 Compat Modes */
-  src: local('Open Sans Regular'), local('OpenSans-Regular'),
-    url('../../fonts/open-sans-v17-latin-regular.eot?#iefix')
-      format('embedded-opentype'),
-    /* IE6-IE8 */ url('../../fonts/open-sans-v17-latin-regular.woff2')
-      format('woff2'),
-    /* Super Modern Browsers */
-      url('../../fonts/open-sans-v17-latin-regular.woff') format('woff'),
-    /* Modern Browsers */ url('../../fonts/open-sans-v17-latin-regular.ttf')
-      format('truetype'),
-    /* Safari, Android, iOS */
-      url('../../fonts/open-sans-v17-latin-regular.svg#OpenSans') format('svg'); /* Legacy iOS */
+  src: local(''),
```

Comment:
Only thing that might be worth keeping is the `local('Open Sans Regular'), local('OpenSans-Regular')` sources. This allows locally installed versions to be used if available.

Reply:
Ah yes, oops!  Thanks